src/game/AI/BaseAI/PetAI.cpp

```diff
@@ -59,13 +51,6 @@ PetAI::PetAI(Creature* creature) : UnitAI(creature), m_creature(creature), inCom
             m_meleeEnabled = false;
             break;
     }
-
-    // TODO: Remove this when unit support is removed
```

Comment:
Unit support is still not removed. Players use this AI when in charm. The code below is still valid.